docs/api/overview.md

```diff
@@ -1,57 +1,248 @@
 ---
+title: API Overview
+sidebar_position: 1
 vega_network: TESTNET
 ---
+
 import Topic from '/docs/topics/_topic-development.mdx'
 import DataNodes from '@site/src/components/DataNodes';
-
-# Using the APIs
+import EthAddresses from '@site/src/components/EthAddresses';
 
 <Topic />
 
-There are a number of ways to interact with Vega through APIs and Ethereum bridge contracts, and below you can learn more about the reference documentation and tutorials that are currently available.
+## Introduction
+The Vega protocol forms a blockchain specifically built to support trading of financial products on markets proposed and voted on by members of the Vega community.
 
-As most of the APIs are designed to use for trading-related queries, the best place to try them out is on the testnet network, also known as Fairground. 
+There are several ways to interact with Vega through APIs and smart contracts, and below you can learn more about the overall technical structure, the reference documentation, and tutorials that are currently available.
 
-## Tutorials
-Tutorials provide the information you'll need about the protocol to understand and use the guide, as well as instructions on how to interact with scripts, API calls, or other code. 
+As most of the APIs are designed to be used for trading-related queries, the best place to try them out is on the testnet network, also known as Fairground. The public endpoints differ between testnet and mainnet, as do the network configurations your Vega-compatible wallet needs in order to connect. See the public endpoints page for details. You can switch between documentation of the networks' software versions in the top menu bar on every page.
 
-**[Tutorials](../tutorials)**: See the tutorial(s) currently available for this network.
+The easiest way to get started with Vega programmatically is by using REST. There are also [gRPC](./grpc/overview.md) and [GraphQL](./graphql/generated.md) endpoints geared towards users with specific requirements. For more about the endpoints they provide, go to their specific sections.
+
+Because Vega is a decentralised network run by independent validators, there are a number of different servers that you'll need to interact while working with the network - data will generally be recieved from one service and transactions will be sent to another. See the [the next section on data flow](#data-flow).
+
+## Data flow
+Most of the data you will want to access will come from the trading data REST service. This is served by a 'data node' - servers that read the blockchain and produce a database of the current state, and in some cases store the past state. These data nodes are where you will read data from. 
+
+To 'write' to the chain - that is to send in a transaction - you will use a Vega wallet, which will sign your transactions and send them to a validator node to be included in the chain.
+
+![How data cycles through the Vega network](/img/concept-diagrams/data-flow-vega.png)
+
```

Comment:
how can we express that core is also readable. 

diagram idea: components of a system, pull the apis from core service, be from be service, etc.